### DISCUSSION TITLE
Led leuchtet nicht bei Bildaufnahme

### DISCUSSION THREAD

**Discussion by xerion8300** [jomjol/AI-on-the-edge-device#1392] [Q&A]
Hallo ich habe das Problem wenn die Kammera hochfährt das die LED leuchte.
Wenn ich aber ein Bild von der Uhr machen möchte wird die LED nicht eingeschaltet, und es bleibt alles schwarz.
Bei LED Intensity habe ich auch schon geändert aber immer das selbe.

**Comment by jomjol** [comment]
Kannst du bitte das Logfile und idealerweise ein Serial-Log schicken. Sonst die die Fehlerdiagnose nicht möglich. Danke!

**Comment by xerion8300** [comment]
[log_2022-11-23.txt](https://github.com/jomjol/AI-on-the-edge-device/files/10073763/log_2022-11-23.txt)

**Comment by jomjol** [comment]
Du müsstest in der Konfiguration erstmal das Logging einschalten.

**Comment by xerion8300** [comment]
So habe mal die Log aktiviert 
Sende noch ein paar Bilder mit hoffe du kannst mir weiterhelfen.
![2022-11-24 09_46_17-Window](https://user-images.githubusercontent.com/25564472/203735545-88dfd4c3-6fca-4d9a-8a16-bdcb6c2373df.png)
![2022-11-24 09_45_17-Window](https://user-images.githubusercontent.com/25564472/203735554-437355e6-74ac-419b-b537-93914eb189f0.png)
![2022-11-24 09_44_52-Window](https://user-images.githubusercontent.com/25564472/203735560-324c5066-bc87-4858-a07c-afb69dd0ac29.png)
![2022-11-24 09_44_16-Window](https://user-images.githubusercontent.com/25564472/203735569-0a74ff53-64de-4f5b-8043-7374ce4eb620.png)

**Comment by xerion8300** [comment]
Die log sagt wieder garnichts
Firmware habe ich die neuste drauf vom 29.9.2022
[log_2022-11-24.txt](https://github.com/jomjol/AI-on-the-edge-device/files/10082279/log_2022-11-24.txt)

**Comment by jomjol** [comment]
Irgendwas stimmt da mit deinem Setup nicht. Die Version vom 29.09.22 kann den Fehler mit "The Version of the Webfile ... " eigentlich  noch gar nicht, da es erst vor kurzem in der Rolling implementiert!

Mach mal ein Screenshot von Info:
![grafik](https://user-images.githubusercontent.com/30766535/203841708-2c35227c-dbc0-49b6-8998-46419634554b.png)

**Comment by xerion8300** [comment]
So Aktueller Screen
![2022-11-25 09_08_00-watermeter - AI on the edge – Mozilla Firefox](https://user-images.githubusercontent.com/25564472/203932174-77bd77fa-48fc-4284-bcc1-0899beff6c67.png)

**Reply by jomjol** [reply]
Da stimmt etwas mit deinem HTML Verzeichnis nicht:
![grafik](https://user-images.githubusercontent.com/30766535/204032062-c6753b02-10e9-4e2b-9ad8-44da329f143f.png)
Das ist ein komplett frisch installiertes Device direkt nach der ersten Konfiguration.
